uk.co.compucorp.civicrm.tasksassignments/js/src/crm-tasks-workflows/controllers/casetype.extended.controller.js

```diff
@@ -26,7 +30,7 @@
      */
     function addActivity (activitySet, activityType) {
       // call parent
```

Comment:
This comment is excessive, the code now is self-explanatory. Please delete this comment.